report/src/bluetooth_presence_methods/fingerprinting.tex

```diff
@@ -0,0 +1,25 @@
+\section{Fingerprinting}
+Fingerprinting is a technique utilizing the measured power of the received Bluetooth signal to accommodate the physical layout of the room in which the system is deployed. 
+Setting up a system utilizing fingerprinting requires two phases: a phase where the layout of the room is analysed to accommodate the impact of physical obstacles in the room, and a phase where the unknown position of a Bluetooth receiver is inferred \cite{presence_ble_review, taking_localization_to_the_wild}.
+
+During the first phase, a measuring device is used to measure RSSIs for known positions.
+The RSSI measurements and the actual position of the devices, as well as any additional features for the model, are collected. 
```

Comment:
Any additional features for the model sounds weird - what are we trying to say here?